watchdog/main.go

```diff
@@ -145,6 +145,8 @@ func pipeRequest(config *WatchdogConfig, w http.ResponseWriter, r *http.Request,
 		if config.writeDebug == true {
 			log.Printf("Success=%t, Error=%s\n", targetCmd.ProcessState.Success(), err.Error())
 			log.Printf("Out=%s\n", out)
+		} else {
+			log.Printf("An Error Occurred\n")
```

Comment:
Remove this line.

Reply:
I just added that as it might be helpful to see that there was an error without sending out too much information.